src/plot.jl

```diff
@@ -64,7 +64,7 @@ const supportedplots = push!(collect(keys(translationdict)), :mixeddensity, :cor
         lags = 0:(maxlag === nothing ? round(Int, 10 * log10(length(range(c)))) : maxlag)
         # Chains are already appended in `c` if desired, hence we use `append_chains=false`
         ac = autocor(c; sections = nothing, lags = lags, append_chains=false)
-        ac_mat = convert(Array, ac)
+        ac_mat = cat(reduce.(hcat, Iterators.drop.(ac, 1))...; dims=3)
```

Comment:
How large is this `...` generally going to be? Could it cause a blow-up in compilation times?

Reply:
I think for many lags, `autocor` itself would actually have a problem, since `SummaryStats`, like `ChainsDataFrame`, is backed by a `NamedTuple` whose entries are the columns.

A safer approach would be to modify PosteriorStats so that `SummaryStats` wraps any Table. Then `autocor` could be modified to produce a `SummaryStats` backed by an `OrderedDict`. 

This `cat` call could be easily replaced by `Compat.stack`.

Reply:
> I think for many lags, autocor itself would actually have a problem, since SummaryStats, like ChainsDataFrame, is backed by a NamedTuple whose entries are the columns.

Gotcha :+1: 

> This cat call could be easily replaced by Compat.stack.

Should we maybe do this regardless of whether `autcorr` suffers from the same issue? This way things will "just work" once we fix PosterioStats:)

Reply:
I started on making `SummaryStats` wrap any Table and decided to make some breaking changes while I'm at it. Once I finish those, I'll update this PR accordingly.

Reply:
Sounds good:)